config/exercise_readme.go.tmpl

```diff
@@ -35,6 +52,7 @@ If you want to know more about Exercism, take a look at the [contribution guide]
 [help-page]: http://exercism.io/languages/rust
 [modules]: https://doc.rust-lang.org/book/second-edition/ch07-00-modules.html
 [cargo]: https://doc.rust-lang.org/book/second-edition/ch14-00-more-about-cargo.html
+[rust-tests]: https://doc.rust-lang.org/book/second-edition/ch11-02-running-tests.html
```

Comment:
Without including a markdown link like this `Documentation about [tests][rust-tests]` elsewhere in the readme, markdown processors will strip this link entirely. Please add such a markdown link somewhere.